Fix console output

elvetemedve opened the pull request:
This is a possible solution to [issue #1](https://github.com/anho/BehatFormatterTeamcity/issues/1).

The class `Behat\Behat\Output\Printer\ConsoleOutputPrinter` has been removed from Behat 3.1, making the extension unable to operate properly.
This commit substitutes that class with a new `ConsoleOutput` class. This class implements the `OutputPrinter` interface, which has not changed between Behat versions. So it makes the extension compatible with Behat 3.x versions. The actual printing to console is carried out by Symfony's `ConsoleOutput` class.

anho commented:
Don't you think we should stick to the possibilities Behat provides out of the box? To be more precise: There is already a bunch of classes inside Behat that is concerned with outputting messages to streams (like the standard output) and I'd rather reuse these instead of adding more code on top of the code base.

elvetemedve commented:
@anho I agree, the best would be to use one of objects from Behat to write to the console (for many reasons). But unfortunately I haven't found such an object which is published through the DI container. 
Maybe this one could work: https://github.com/Behat/Behat/blob/master/src/Behat/Testwork/Cli/Application.php#L178
I'll check it later.

elvetemedve commented:
@anho I've removed the Symfony console output dependency from Composer and replaced it with the `cli.output` service provided by Behat.
Please check the PR again!

anho commented:
@elvetemedve sorry for coming back so late, but nevertheless many thanks, good work.
